tools/dialignr/dialignr.xml

```diff
@@ -0,0 +1,51 @@
+<tool id="dialignr" name="DIAlignR" version="@WRAPPER_VERSION@">
+    <description>for retention time alignment of targeted mass spectrometric data</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <requirements>
+        <requirement type="package" version="@TOOL_VERSION@">bioconductor-dialignr</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    mkdir -p ./data/mzml &&
+    mkdir -p ./data/osw &&
+
+    #for $mzml in $input_mzml_files:
+        #set $file_name_mzml = $mzml.element_identifier.split(".")
```

Comment:
you will need to sanitize the identifier like here: https://github.com/galaxyproject/tools-iuc/blob/2808c3c3e47e43eaf835d725fa64c63ee6dc0913/tools/chromeister/chromeister.xml#L13 otherwise its not safe to use for the link name. 

Since in Galaxy users can chose to have any characters in the identifiers (including `'.'`) `split(".")` might not be a good idea. Anyway in real life there will not be an extension in the identifier (its only in the tests where the identifier is equal to the uploaded file name).

Reply:
Thanks for pointing that out! For the sake of simplicity, I just removed the extension of the test files.